src/PrestaShopBundle/Resources/views/Admin/Improve/International/Translations/translations_settings.html.twig

```diff
@@ -47,4 +45,4 @@
   {{ parent() }}
 
   <script src="{{ asset('themes/new-theme/public/translation_settings.bundle.js') }}"></script>
-{% endblock %}
+{% endblock %}
```

Comment:
Please add an empty line to the end of the file (you can automate it in your VSCode configuration)

Reply:
Done and configured in VS code ;-)